fees/defiplaza/index.ts

```diff
@@ -0,0 +1,73 @@
+import { FetchResultGeneric, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { getChainVolume } from "../../helpers/getUniSubgraphVolume";
+import fetchURL from "../../utils/fetchURL";
+
+type RadixPlazaResponse = {
+	date: number,
+	stateVersion: number,
+	totalValueLockedUSD: number,
+	volumeUSD: number,
+	feesUSD: number,
+	royaltiesUSD: number,
+	swaps: number
+}
+
+const thegraph_endpoints = {
+	[CHAIN.ETHEREUM]: "https://api.thegraph.com/subgraphs/name/omegasyndicate/defiplaza"
+};
+const radix_enpoint = "https://radix.defiplaza.net/api/defillama/volume";
+
+const graphs = getChainVolume({
+	graphUrls: thegraph_endpoints,
+	totalVolume: {
+		factory: "factories",
+		field: "totalTradeVolumeUSD",
+	},
+	dailyVolume: {
+		factory: "dailie",
+		field: "tradeVolumeUSD",
+		dateField: "date"
+	},
+});
+
+const adapter: SimpleAdapter = {
+	adapter: {
+		[CHAIN.ETHEREUM]: {
```

Comment:
is there fees detail for eth chain ? now fees eth chain does not return value like fees `RADIXDLT` chain `dailyFees`
`dailyRevenues`

Reply:
That is a good point! I just added it to our subgraph, and it is syncing.

I will update the PR once we have the `dailyFees` for Eth chain. We don't have any `dailyRevenue` though, so I will _not_ add those as 0, right.